android/src/main/kotlin/com/onegini/mobile/sdk/flutter/helpers/ResourceHelper.kt

```diff
@@ -1,31 +1,45 @@
 package com.onegini.mobile.sdk.flutter.helpers
 
 import com.google.gson.Gson
+import com.onegini.mobile.sdk.flutter.OneWelcomeWrapperErrors
 import com.onegini.mobile.sdk.flutter.OneWelcomeWrapperErrors.*
+import com.onegini.mobile.sdk.flutter.OneginiSDK
 import com.onegini.mobile.sdk.flutter.constants.Constants.Companion.RESPONSE_BODY
 import com.onegini.mobile.sdk.flutter.constants.Constants.Companion.RESPONSE_HEADERS
 import com.onegini.mobile.sdk.flutter.constants.Constants.Companion.RESPONSE_STATUS_CODE
 import com.onegini.mobile.sdk.flutter.constants.Constants.Companion.RESPONSE_URL
+import com.onegini.mobile.sdk.flutter.pigeonPlugin.HttpRequestMethod.GET
+import com.onegini.mobile.sdk.flutter.pigeonPlugin.HttpRequestMethod.POST
+import com.onegini.mobile.sdk.flutter.pigeonPlugin.HttpRequestMethod.PUT
+import com.onegini.mobile.sdk.flutter.pigeonPlugin.HttpRequestMethod.PATCH
+import com.onegini.mobile.sdk.flutter.pigeonPlugin.HttpRequestMethod.DELETE
```

Comment:
Seems like most of added imports are not used.

Reply:
forgot to remove this file;